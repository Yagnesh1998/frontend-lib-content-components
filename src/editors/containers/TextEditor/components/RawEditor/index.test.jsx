import React from 'react';
import { shallow } from 'enzyme';

import { RawEditor } from '.';

describe('ImageSettingsModal', () => {
  const props = {
    editorRef: {
      current: {
        value: 'Ref Value',
      },
    },
    text: 'sOmErAwHtml',
  };
  test('renders as expected with default behavior', () => {
    expect(shallow(<RawEditor {...props} />)).toMatchSnapshot();
  });
});
