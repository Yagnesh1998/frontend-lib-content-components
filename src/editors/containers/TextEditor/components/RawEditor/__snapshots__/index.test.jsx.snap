// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageSettingsModal renders as expected with default behavior 1`] = `
<div
  className="form-group"
  style={
    Object {
      "padding": "10px 30px",
    }
  }
>
  <Alert
    variant="danger"
  >
    You are using the raw HTML editor.
  </Alert>
  <textarea
    className="form-control"
    rows="12"
  >
    sOmErAwHtml
  </textarea>
</div>
`;
